examples/manually_deploy_all_applications.sh

```diff
@@ -13,6 +13,13 @@ pause(){
     read -p "Press [Enter] key to continue..."
 }
 
+check_variable_is_set(){
+    if [[ -z ${!1} ]]; then
+        echo "$1 must be set and non empty. ($2)"
+        exit 1
+    fi
+}
+
```

Comment:
Missed from the original script - I must have `source`d cf_function.sh when I ran it last time.